Use official SQLite framework

milgner opened the pull request:
This replaces the bundled SQLite sources with a reference to the official SQLite3 WinRT package, which should facilitate maintenance in the future.
I also added a few lines to the README on how to integrate the project in a WinJS application.

Commit:
Replace bundled SQLite with reference to official package

Commit:
Update README with some setup instructions

Commit:
Rename native component to `SQLite3Wrapper`

This is to avoid naming conflicts in generated DLL files with the
new external dependency.

superquadratic commented:
For some reason, the default active platform (selected when you first open the solution) is ARM. As a result, the test app (SQLite3JS) cannot be run locally. The active platform must be changed in the Configuration Manager (!) to get it to run on the local machine.

I think this is a little unfortunate. Maybe there is a way to make it default to something "native" that can be run locally.
@milgner, did you change the `*ConfigurationPlatforms` sections in the solution file or did Visual Studio do that automatically?

milgner commented:
I removed some platforms, because with the binary component, 'Mixed Platform' or 'Any CPU' didn't make much sense to me. Maybe we should remove ARM, too, since we have no possibility to test it at the moment and then add it back as soon as we do?

superquadratic committed:
Remove untested ARM platform

Since we have no means of testing the ARM platform, we removed it for now.
It was also the default platform, which was a little unfortunate, since it
prevented the test app to run on the local machine without manually
changing the platform.
